test/cbmc/proofs/ota_http/OTA_HTTP_strerror/Makefile

```diff
@@ -0,0 +1,26 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+HARNESS_ENTRY = OTA_HTTP_strerror_harness
+HARNESS_FILE = $(HARNESS_ENTRY)
+
+# This should be a unique identifier for this proof, and will appear on the
+# Litani dashboard. It can be human-readable and contain spaces if you wish.
+PROOF_UID = OTA_HTTP_strerror
+
+DEFINES +=
+INCLUDES += $(SRCDIR)/source/include
+
+REMOVE_FUNCTION_BODY +=
+UNWINDSET +=
+
+PROOF_SOURCES += $(PROOFDIR)/$(HARNESS_FILE).c
+PROJECT_SOURCES += $(SRCDIR)/source/ota_http.c
+
+# If this proof is found to consume huge amounts of RAM, you can set the
+# EXPENSIVE variable. With new enough versions of the proof tools, this will
+# restrict the number of EXPENSIVE CBMC jobs running at once. See the
+# documentation in Makefile.common under the "Job Pools" heading for details.
+# EXPENSIVE = true
+
+include ../../Makefile.common
```

Comment:
Wouldn't this be the `root/test/cbmc/proofs/ota_http/` directory? I don't see a "Makefile.common" file there.

Reply:
No. Since the makefile is in `root/test/cbmc/proofs/ota_http/OTA_HTTP_strerror/ ` it would go back to `root/test/cbmc/proofs/ota_http/`

Reply:
I see, you are correct. I checked that file and it seems like it's just defined to a path. I went to that path and the example common makefile looked quite different. Is this Makefile.common file supposed to be updated?

Reply:
It's not supposed to be updated but it might have changed from the previous version of the template. I'll look into it.

Reply:
There seem to no changes to this `Makefile,common` from the example `Makefile.common` .